.gitpod.yml

```diff
@@ -1,5 +1,6 @@
 tasks:
-  - init: ./mvnw verify spotless:apply -DskipTests=true
+  - init: sdk install mvnd
```

Comment:
shouldn't installing mvnd go to `before` task and `mvnd verify spotless:apply -DskipTests=true` to `init`?

Reply:
ah! yes.
had forgotten about the `before` section in the config.
made the changes.